docs/_static/robusta

```diff
@@ -35,8 +35,8 @@ fi
 if [[ "upgrade-cli" == "$COMMANDS" ]]; then
   docker pull us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli > /dev/null 2>&1 & spinner
   echo 'Upgrade complete'
-  docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli robusta version
+  docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli version
```

Comment:
Is the name of the command missing here?
Do we need 
```
-v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli  robusta-cli version
```

Reply:
no, we don't need it as the new Dockerfile uses a python entrypoint